src/runtime/runtime.js

```diff
@@ -709,6 +721,29 @@ export class ConfiguredRuntime {
   /** @override */
   showOffers(opt_options) {
     return this.documentParsed_.then(() => {
+      if (opt_options['oldSku']) {
+        console.error(
```

Comment:
Is this the sort place where I should use assert as well, or is the current structure fine?

Reply:
yes any place you generate a console error

Reply:
@dparikh ?

Reply:
Instead of console.error() use Throw Error("msg")

Reply:
It'd rather we use `assert()` over `throw New Error()` to generate the Error because it localizes the error path through the logger so we can log and capture errors if we choose to.

Reply:
I went ahead and turned all instances of `throw new Error` in runtime into `assert()` instead.